stock_manual_transfer/views/stock_route_views.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">stock.route</field>
         <field name="inherit_id" ref="stock.stock_location_route_form_view" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='warehouse_selectable']" position="after">
+            <xpath expr="//div[field[@name='warehouse_ids']]" position="after">
```

Comment:
For readability, I think it would be better to reference the selectable field:

```suggestion
            <xpath expr="//div[field[@name='warehouse_selectable']]" position="after">
```

(I had to check original code to understand this)